### ISSUE TITLE
SiteMap Editor Update Area Icon

### ISSUE THREAD

**Issue by soziblewuup** [MscrmTools/XrmToolBox#251]
Hi, 

I can't seem to change the Icon of an Area using the SiteMap Editor.

I've tried using the paths Icon=____, Icon="____", and just ____ on it's own and still no luck updating the Icons. 
(____ being the web resource)

Does you have any specific information about how to use this / what the filetype and Icon size limitations etc. are. I'm connected to an instance of 2015 Online Update 1. 

Thanks :).

**Comment by MscrmTools** [comment]
the format should be $webresource:______, if you use the web resource selector, it should add the correct format

**Comment by soziblewuup** [comment]
Thanks for the response. I've tried and still no luck. I've tried multiple web resource file types and multiple image sizes (16 x 16, 32 x 32, 85 x 79). 

I've attached two images below to show you the process I'm going through. Perhaps I'm doing something incorrect. 

Thanks!

![screen shot 2015-09-24 at 15 08 25](https://cloud.githubusercontent.com/assets/14819631/10074210/c4404ea8-62ce-11e5-8d12-4100be433010.png)

![screen shot 2015-09-24 at 15 12 17](https://cloud.githubusercontent.com/assets/14819631/10074214/c7d8b820-62ce-11e5-9558-ea121b9d6e28.png)
